assets/js/reducers/guest.js

```diff
@@ -34,15 +34,25 @@ const validEmail = (email) => {
 }
 
 const changeEmail = (state, action) => {
+  const { email, collaborators } = action;
```

Comment:
Aren't the `collaborators` part of the `state` already? Since it's not data generated by the user's action, I don't think we should be sending it in the action itself - but rather get it from the `state`.